spec/unit/provider/oneview_volume_c7000_spec.rb

```diff
@@ -18,7 +18,7 @@
 require_relative '../../support/fake_response'
 require_relative '../../shared_context'
 
```

Comment:
line 85 of 'lib/puppet/provider/oneview_volume/c7000.rb' is not being touched by this file. 
Should add a ``get_snapshot`` which has a ``name`` so that we reach 100% path coverage